src/test/java/com/google/api/codegen/protoannotations/testdata/java_library_no_gapic_config.baseline

```diff
@@ -1070,7 +1070,7 @@ public class LibraryServiceClient implements BackgroundResource {
    * Sample code:
    * <pre><code>
    * try (LibraryServiceClient libraryServiceClient = LibraryServiceClient.create()) {
-   *   BookName name = BookName.of("[BOOK_SHELF]", "[BOOK]");
```

Comment:
`Book` is a multi-pattern resource. Without gapic v2, the generator does not know how to instantiate an instance of it, so we use a string literal directly.